src/Tribe/Utils/Element_Attributes.php

```diff
@@ -0,0 +1,228 @@
+<?php
+namespace Tribe\Utils;
+
+/**
+ * Class Element_Attributes to handle HTML attributes for elements.
+ *
+ * @since  TBD
+ *
+ * @package Tribe\Utils
+ */
+class Element_Attributes {
+	/**
+	 * Store the results of parsing the attributes.
+	 *
+	 * @since  TBD
+	 *
+	 * @var array
+	 */
+	protected $results = [];
+
+	/**
+	 * Stores the arguments passed.
+	 *
+	 * @since  TBD
+	 *
+	 * @var array
+	 */
+	protected $arguments = [];
+
+	/**
+	 * Setups an instance of Element Attributes.
+	 *
+	 * @since  TBD
+	 *
```

Comment:
Add a note here about the function parameters.

```suggestion
	 * @param string $element One or more elements to build the validated and sanitized attributes. 
```